ioet_feature_flag/strategies/pilot_users.py

```diff
@@ -0,0 +1,25 @@
+import typing
+
+from .strategy import Strategy
+from ..exceptions import MissingToggleAttributes
+from ..toggle_context import ToggleContext
+
+
+class PilotUsers(Strategy):
+    def __init__(self, enabled: bool, allowed_users: typing.List[str]) -> None:
```

Comment:
Would you mind changing the type annotation for `allowed_users`? On the test for this class I saw that the allowed users attribute is a str

Reply:
oops it was actually supposed to be a list, but I forgot to split the string